Makefile

```diff
@@ -6,7 +6,7 @@
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-SWAGGER := docker run --rm -it -e GOPATH=$(shell go env GOPATH):/go -v $(HOME):$(HOME) -w $(shell pwd) quay.io/goswagger/swagger:v0.29.0
+SWAGGER := docker run --rm -it -e GOPATH=$(shell go env GOPATH):/go -v $(HOME):$(HOME) -w $(shell pwd) quay.io/goswagger/swagger:v0.30.3
```

Comment:
I would prefer pinning go-swagger to a specific version, so that we are sure everybody is using the same container to generate the client code. Otherwise, it might happen that different people use different versions because they have pulled the docker image at different times.

Reply:
We could use something like `:latest` to make sure everyone will use the newest version all the time. I'm a bit afraid that we will forget to unpin it in the future and miss the latest updates from the project